src/formats/TNG.cpp

```diff
@@ -45,6 +45,11 @@ TNGFormat::TNGFormat(std::string path, File::Mode mode, File::Compression compre
     if (compression != File::DEFAULT) {
         throw format_error("TNG format do not support compression");
     }
+
+    int64_t exp = -1;
+    CHECK(tng_distance_unit_exponential_get(tng_, &exp));
```

Comment:
Is this guaranteed to be set in tng files?

Reply:
`tng_io` [says](https://github.com/chemfiles/tng/blob/c44bdcd5554d498aed93527e9f7c5c1f77b160cd/src/lib/tng_io.c#L296-L299) it is using nm as default. The corresponding value is set [here](https://github.com/chemfiles/tng/blob/c44bdcd5554d498aed93527e9f7c5c1f77b160cd/src/lib/tng_io.c#L8788) when initializing a new file. So this value should be set always.